stablehlo/tests/legalize_stablehlo_to_vhlo.mlir

```diff
@@ -399,24 +399,24 @@ func.func @op_all_to_all(%arg0: tensor<4x16xf32>) -> tensor<16x4xf32> {
 // CHECK-LABEL: "op_all_to_all"
 
 func.func @op_and(%arg0: tensor<i1>, %arg1: tensor<i1>) -> tensor<i1> {
-  // CHECK: "vhlo.and"(%arg0, %arg1) : (tensor<i1>, tensor<i1>) -> tensor<i1>
+  // CHECK: "vhlo.and"(%arg0, %arg1) : (!vhlo.tensor<!vhlo.integer<i1>>, !vhlo.tensor<!vhlo.integer<i1>>) -> !vhlo.tensor<!vhlo.integer<i1>>
   %0 = "stablehlo.and"(%arg0, %arg1) : (tensor<i1>, tensor<i1>) -> tensor<i1>
   func.return %0 : tensor<i1>
 }
 // CHECK-LABEL: "op_and"
 
 func.func @op_atan2(%arg0: tensor<f32>, %arg1: tensor<f32>) -> tensor<f32> {
-  // CHECK: "vhlo.atan2"(%arg0, %arg1) : (tensor<f32>, tensor<f32>) -> tensor<f32>
+  // CHECK: "vhlo.atan2"(%arg0, %arg1) : (!vhlo.tensor<!vhlo.f32>, !vhlo.tensor<!vhlo.f32>) -> !vhlo.tensor<!vhlo.f32>
   %0 = "stablehlo.atan2"(%arg0, %arg1) : (tensor<f32>, tensor<f32>) -> tensor<f32>
   func.return %0 : tensor<f32>
 }
 // CHECK-LABEL: "op_atan2"
 
 func.func @op_batch_norm_grad(%arg0: tensor<16x16x16x16xf32>, %arg1: tensor<16xf32>, %arg2: tensor<16xf32>, %arg3: tensor<16xf32>, %arg4: tensor<16x16x16x16xf32>) -> (tensor<16x16x16x16xf32>, tensor<16xf32>, tensor<16xf32>) {
   //      CHECK: "vhlo.batch_norm_grad"(%arg0, %arg1, %arg2, %arg3, %arg4) {
-  // CHECK-SAME:   epsilon = 1.000000e-03 : f32,
-  // CHECK-SAME:   feature_index = 0 : i64
-  // CHECK-SAME: } : (tensor<16x16x16x16xf32>, tensor<16xf32>, tensor<16xf32>, tensor<16xf32>, tensor<16x16x16x16xf32>) -> (tensor<16x16x16x16xf32>, tensor<16xf32>, tensor<16xf32>)
+  // CHECK-SAME:   epsilon = #vhlo.float<1.000000e-03 : !vhlo.f32>,
+  // CHECK-SAME:   feature_index = #vhlo.integer<0 : i64>
```

Comment:
Float is forked, integer is wrapped. I'm thinking I'll wrap float as well, because I'm not sure I'm doing the float conversions correctly, and wrapping would prevent us from needing to worry about that. Would just take a bit of extra logic in the bytecode writer.